.golangci.yml

```diff
@@ -0,0 +1,13 @@
+linters-settings:
+  goconst:
+    # minimal length of string constant, 3 by default
+    min-len: 3
+    # minimal occurrences count to trigger, 3 by default
+    min-occurrences: 5
```

Comment:
What exactly does these two choices mean?

Reply:
Comments are taken from the original documentation. The first one is the minimal length of the string that can be considered for a constant and the second one is the minimal number of that string occurrences in the package to be reported as constant definition required.